abstract.md

```diff
@@ -0,0 +1,41 @@
+# Making research software more robust
+Morgan Taschuk and Greg Wilson
+
+
+At the cutting edge of research, bioinformatics software is often written by a
+single person who is both developer and scientist, producing insightful, novel
+results for their own projects. Upon presenting their work, other
+people will want to use their methods. But much of academic software is written
+in a prototypic, exploratory and rigid style that is incompatible with reuse. Often, the end user will
+need to spend a considerable amount of time either reading code or in
+communication with the original developers. More times than we like to admit,
+we will simply give up on the software, either writing our own or finding
+another that accomplishes the same purpose. This practice is detrimental to
+creating reproducible results, to the pace of scientific research, and to
+creating bioinformatics communities that build on each other's work.
+
+This familiar problem has spawned a range of solutions, from virtual machines
+and containerization[1], to more training for bioinformaticians
+[2], to reform of scientific reward [3]. All of
+these solutions require time and money, and none directly contribute to
+scientific advancement. What can the front-line bioinformatician do right now
```

Comment:
I would cut the rhetorical question and use, "As an alternative, we present a set of guidelines..." (joining up the two paras)